assets/src/components/conversations/ConversationHeader.tsx

```diff
@@ -114,12 +117,20 @@ const ConversationHeader = ({
 
           <Box mx={1}>
             {status === 'closed' ? (
-              <Tooltip title="Reopen conversation" placement="bottomRight">
-                <Button
-                  icon={<UploadOutlined />}
-                  onClick={() => onReopenConversation(conversationId)}
-                />
-              </Tooltip>
+              <Fragment>
+                <Tooltip title="Reopen conversation" placement="bottomRight">
+                  <Button
+                    icon={<UploadOutlined />}
+                    onClick={() => onReopenConversation(conversationId)}
+                  />
+                </Tooltip>
+                <Tooltip title="Delete conversation" placement="bottomRight">
```

Comment:
to fix the spacing issue i mentioned in another comment, probably just need to render this in another `<Box mx={1}>` that only gets shown when `status === 'closed'`